tools/doc/type-parser.js

```diff
@@ -52,6 +52,7 @@ const customTypesMap = {
   'Cipher': 'crypto.html#crypto_class_cipher',
   'Decipher': 'crypto.html#crypto_class_decipher',
   'DiffieHellman': 'crypto.html#crypto_class_diffiehellman',
+  'errors.Error': 'errors.html#errors_class_error',
```

Comment:
IIUC, the types are groupped by the lib modules here (module groups are sorted, then types are sorted inside their group), so this entry need to init its own group between the `domain.html` and `esm.html` groups.